opendut-types/src/util/net.rs

```diff
@@ -51,3 +51,32 @@ pub enum NetworkInterfaceNameError {
     #[error("Due to operating system limitations, the name for network interfaces may not be longer than {max} characters!")]
     TooLong { value: String, max: usize }
 }
+
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct Certificate(pub String);
```

Comment:
Maybe we can replace our new-type with a 'real' certificate type for example from rustls.